python/examples/advanced/Segments.ipynb

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
This looks like it's just the test code without any example or anything. Is this a temporary situation? What's the benefit between putting this in vs having Felipe or I make an example on Monday that is easy to follow?

Reply:
I can leave it out, or we can update it, just thought showcasing how to run it would help, I actually used this to test upload to whylabs etc, but left those out of the PR.